packages/amplify-category-auth/provider-utils/awscloudformation/assets/cognito-defaults.js

```diff
@@ -99,7 +99,9 @@ const entityKeys = {
 };
 
 const getAllDefaults = (name) => {
-  const projectName = name.projectConfig ? `${name.projectConfig.projectName.toLowerCase()}${sharedId}` : name;
+  const disallowedChars = /[^A-Za-z0-9_]+/g;
```

Comment:
This doesn't belong to this PR.

Reply:
This change is gone now after merging with upstream/multienv